app/assets/javascripts/analytics.js

```diff
@@ -88,6 +88,15 @@ Blacklight.onLoad(function(){
     }
   });
 
+  // HathiTrust Temporary Access
+  $('.panel.temporary-access a, .results-online-section .online-links').on('click', function(e) {
+    if (/Full text via HathiTrust/.test(this.text)) {
+      ga('send', 'event', 'HathTrust', 'clicked', this.href, {
```

Comment:
Small typo (or brevity)? 

```suggestion
      ga('send', 'event', 'HathiTrust', 'clicked', this.href, {
```

Reply:
![](https://media.giphy.com/media/9DJtFRgk0tOla/giphy.gif)